src/main/java/org/prgrms/kdt/voucher/model/VoucherType.java

```diff
@@ -0,0 +1,42 @@
+package org.prgrms.kdt.voucher.model;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum VoucherType {
+    FIXED(1, "FixedAmountVoucher", "Type your discount amount"),
+    PERCENT(2, "PercentDiscountVoucher", "Type your discount percent");
+
+    VoucherType(int voucherTypeNum, String voucherTypeName, String benefitMessage) {
+        this.voucherTypeNum = voucherTypeNum;
+        this.voucherTypeName = voucherTypeName;
+        this.benefitMessage = benefitMessage;
+    }
+
+    private final int voucherTypeNum;
+    private final String voucherTypeName;
+    private final String benefitMessage;
+
+    private static final Map<Integer, VoucherType> voucherTypeNameMap = new HashMap<>();
+    static {
+        for(VoucherType voucherType : VoucherType.values()) {
+            voucherTypeNameMap.put(voucherType.voucherTypeNum, voucherType);
+        }
```

Comment:
p4;
```suggestion
voucherTypeNameMap = Stream.of(VoucherType.values())
            .collect(Collectors.toMap(VoucherType::getVoucherTypeNum, Function.identity()));
        }
```
요런식으로 stream을 활용하면 좀 더 간결하게 나타내줄 수 있을 것 같아요.
참고만 해주고 연습하시면 좋을 것 같아요!!

Reply:
스트림 사용이 미흡해 제대로 사용하지 못했던 것 같습니다 ! 조언 감사드립니다 !!